src/MSBuild/Resources/Strings.resx

```diff
@@ -1235,6 +1235,15 @@
       LOCALIZATION: The prefix "MSBUILD : error MSBxxxx:" should not be localized.
     </comment>
   </data>
+  <data name="InvalidLowPriorityValue" UESanitized="true" Visibility="Public">
+    <value>MSBUILD : error MSB1064: Low priority value is not valid. {0}</value>
```

Comment:
Is the MSB code worth documenting anywhere?

Reply:
I'd ask @ghogen, but my personal opinion is "only if we have a list of error codes for invalid switches already." It took a long time for anyone to even see this, and when someone did, they had intentionally introduced an error. So documentation for this error is "low priority" 😉